cmd/manager.go

```diff
@@ -605,7 +601,9 @@ func logStarted(ctx context.Context, s *jobqueue.ServerInfo, token []byte) {
 	}
 }
 
-func startJQ(ctx context.Context, postCreation []byte) {
+func startJQ(postCreation []byte) {
+	ctx := context.Background()
```

Comment:
If this is only used in the clog.ContextWithFileHandler call, move it inside there to avoid confusion.

Reply:
It's also passed to cloud.New (line 719) that's why I kept it as a variable

Reply:
Ok, fair enough. I'll do the full review now.